tweag/asterius
# Remove a lot of code without breaking stuff

Our `bytestring`/`text` support is now feature-complete in both node/browser. Stream codec works, latin-1 works, thanks to C support.

* Remove `base` md5 runtime shims
* Remove `bytestring` runtime shims
* Remove `text` runtime shims
* Remove `hashable` FNV hash runtime shims
* Remove in-tree `text` package, now we boot with vanilla `text`
* Also fixes `ghc-testsuite` `node` arguments which changed in #804
* Debug mode test `stdout`/`stderr` are discarded for now to avoid jamming up the CI logs
* `nomain` is skipped on CI for now

Should close #758 as well, but the adjustments in `asterius-prelude` and `cloudflare-new` test should come later.
